contrib/refresh_checksums.mk

```
@@ -0,0 +1,83 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+# Invoke this with no arguments to refresh all tarballs, or with a project name to refresh only that project.
+#
+# Example:
+#   make -f contrib/refresh_checksums.mk gmp
+
+SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+JULIAHOME := $(abspath $(SRCDIR)/..)
+
+# Default target that will have everything else added to it as a dependency
+all:
+
+# Get this list via:
+#    using BinaryBuilder
+#    print("TRIPLETS=\"$(join(sort(triplet.(BinaryBuilder.supported_platforms(;experimental=true))), " "))\"")
+TRIPLETS=aarch64-apple-darwin aarch64-linux-gnu aarch64-linux-musl armv6l-linux-gnueabihf armv6l-linux-musleabihf armv7l-linux-gnueabihf armv7l-linux-musleabihf i686-linux-gnu i686-linux-musl i686-w64-mingw32 powerpc64le-linux-gnu x86_64-apple-darwin x86_64-linux-gnu x86_64-linux-musl x86_64-unknown-freebsd x86_64-w64-mingw32
+CLANG_TRIPLETS=$(filter %-darwin %-freebsd,$(TRIPLETS))
+NON_CLANG_TRIPLETS=$(filter-out %-darwin %-freebsd,$(TRIPLETS))
+
+# These are the projects currently using BinaryBuilder; both GCC-expanded and non-GCC-expanded:
+BB_PROJECTS=mbedtls libssh2 nghttp2 mpfr curl libgit2 pcre libuv unwind dsfmt objconv p7zip zlib suitesparse openlibm
+BB_GCC_EXPANDED_PROJECTS=openblas
+BB_CXX_EXPANDED_PROJECTS=gmp llvm
+# These are non-BB source-only deps
+NON_BB_PROJECTS=patchelf mozillacert lapack
+
+# Convert `openblas` to `OPENBLAS`
+define upper
+$(shell echo $(1) | tr '[:lower:]' '[:upper:]')
```

```suggestion
$(shell echo $(1) | tr 'a-z' 'A-Z')
```
You aren't actually changing lower to upper in a user locale, but just changing ASCII chars